lib/semian/redis_client.rb

```diff
@@ -33,19 +33,25 @@ def initialize(semian_identifier, *args)
   CircuitOpenError = Class.new(SemianError)
 
   module SemianConfig
-    attr_reader :raw_semian_options
-
     def initialize(semian: nil, **kwargs)
       super(**kwargs)
 
       @raw_semian_options = semian
     end
 
+    def raw_semian_options
+      @raw_semian_options.respond_to?(:call) ? @raw_semian_options.call(host, port) : @raw_semian_options
```

Comment:
so that was talked about a bit and how it was originally, we added this because now it matches the documentation on how to use the dynamic feature 1:1, while we don't have a need for host and port in our use case, at least now things are consistent across the docs and adapters.